<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>411</width>
      <height>291</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>411</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>CanSat GRIFON - Copy.jpg</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>421</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0">
      <widget class="QLabel" name="packetsReceivedLabel">
       <property name="text">
        <string>Packets Received</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="packetsReceivedLcdNumber">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="digitCount">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="packetsLostLabel">
       <property name="text">
        <string>Packets Lost</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="connectionLabel">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Disconnected</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="connectionStatusLabel">
       <property name="text">
        <string>Connection Status</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLCDNumber" name="packetsLostLcdNumber">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="digitCount">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="missionTimeLabel_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="softwareStateLabel">
       <property name="text">
        <string>Software State</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="missionTimeLabel">
       <property name="text">
        <string>Mission Time</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="stateLabel">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Booth</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QChartView" name="plot1" native="true">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="plot3" native="true">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>340</y>
      <width>521</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="csvTableView">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>680</y>
      <width>1061</width>
      <height>291</height>
     </rect>
    </property>
    <property name="autoScrollMargin">
     <number>5</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QChartView" name="plot2" native="true">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>10</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="plot4" native="true">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>340</y>
      <width>521</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>421</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="connectButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,   stop:0 rgba(60, 186, 162, 255), stop:1 rgba(98, 211, 162, 255))
}
QPushButton {
     background-color: #27f900; border: 1px solid black;
     border-radius: 5px;
}

QPushButton:disabled {
    background-color: rgb(170, 170, 127)
}</string>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disconnectButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,   stop:0 rgb(194, 76, 76), stop:1 rgb(200, 80, 80))
}
QPushButton {
     background-color: #db1818; border: 1px solid black;
     border-radius: 5px;
}

QPushButton:disabled {
    background-color: rgb(194, 76, 76)
}</string>
         </property>
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="powerOff">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: rgb(48, 50, 52)
}
QPushButton {
	color: rgb(255, 255, 255);
     background-color: #000000; border: 1px solid black;
     border-radius: 5px;
}

QPushButton:disabled {
    background-color: rgb(255, 255, 255)
}</string>
         </property>
         <property name="text">
          <string>Power Off</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="startDataRecordingButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,   stop:0 rgba(60, 186, 162, 255), stop:1 rgba(98, 211, 162, 255))
}
QPushButton {
     background-color: #27f900; border: 1px solid black;
     border-radius: 5px;
}

QPushButton:disabled {
    background-color: rgb(170, 170, 127)
}</string>
         </property>
         <property name="text">
          <string>Start Data Recording</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pauseDataRecordingButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,   stop:0 rgb(255, 207, 0), stop:1 rgb(250, 217, 4))
}
QPushButton {
     background-color: #FFFF00; border: 1px solid black;
     border-radius: 5px;
}

QPushButton:disabled {
    background-color: rgb(54, 117, 158)
}</string>
         </property>
         <property name="text">
          <string>Pause Data Recording</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopDataRecordingButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,   stop:0 rgb(194, 76, 76), stop:1 rgb(200, 80, 80))
}
QPushButton {
     background-color: #db1818; border: 1px solid black;
     border-radius: 5px;
}

QPushButton:disabled {
    background-color: rgb(194, 76, 76)
}</string>
         </property>
         <property name="text">
          <string>Stop Data Recording</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="resetPitchAngleButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,   stop:0 rgb(54, 117, 158), stop:1 rgb(60, 115, 150))
}
QPushButton {
     background-color: #0d89db; border: 1px solid black;
     border-radius: 5px;
}

QPushButton:disabled {
    background-color: rgb(54, 117, 158)
}</string>
         </property>
         <property name="text">
          <string>Reset Pitch Angle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetRollAngleButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,   stop:0 rgb(54, 117, 158), stop:1 rgb(60, 115, 150))
}
QPushButton {
     background-color: #0d89db; border: 1px solid black;
     border-radius: 5px;
}

QPushButton:disabled {
    background-color: rgb(54, 117, 158)
}</string>
         </property>
         <property name="text">
          <string>Reset Roll Angle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetAltitudeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,   stop:0 rgb(54, 117, 158), stop:1 rgb(60, 115, 150))
}
QPushButton {
     background-color: #0d89db; border: 1px solid black;
     border-radius: 5px;
}

QPushButton:disabled {
    background-color: rgb(54, 117, 158)
}</string>
         </property>
         <property name="text">
          <string>Reset Altitude</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>QtCharts</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>powerOff</sender>
   <signal>clicked()</signal>
   <receiver>powerOff</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>720</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>720</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>powerOff</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>735</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>696</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
